doc/Using-Omnisharp.md

```diff
@@ -21,10 +23,10 @@ When starting the server you must specify a solution file, or a directory where
 
     OmniSharp.exe -s <solutionPath>
 
-The server will detect the project and start up the project systems for the types of projects it finds.
+The server will detect the project and start up the project systems for the types of projects it finds. There's no way of loading new projects after the server has started.
 
 ## Verbose logging
-You can turn on verbose logging with the `-v` switch.
+You can turn on verbose logging with the `-v` switch, and debug with `-l debug`.
```

Comment:
The `-v` switch is equivalent to `-l debug`

Reply:
Updated